### ISSUE TITLE
Framedrops since the "remove at least 1f of input delay globally #1783 (blu-dev)" patch

### ISSUE THREAD

**Issue by Hagebuttentee** [HDR-Development/HewDraw-Remix#1813]
It is pretty much what it sounds like. Since this patch the vsync works in a different way which causes the game to have framedrops.

Yes, I already posted this on the installation page, but I just found out now that you are supposed to post issues here. (Maybe reading the readme would have helped me, oops.)

**Comment by Hagebuttentee** [comment]
It is pretty much what it sounds like. Since this patch the vsync works in a different way which causes the game to have framedrops.

Yes, I already posted this on the installation page, but I just found out now that you are supposed to post issues here. (Maybe reading the readme would have helped me, oops.)

**Comment by Z0niC101** [comment]
The frame drops are purely visually, iirc blujay said si himself.

**Comment by CSharpM7** [comment]
It's just visual skips but for this issue, would you mind telling us how to reproduce? What stage was it on, what characters, what moves were used...etc

**Comment by Hagebuttentee** [comment]
> It's just visual skips but for this issue, would you mind telling us how to reproduce? What stage was it on, what characters, what moves were used...etc

Honestly it happens pretty much all the time, I couldn't name specific scenarios. Also sorry for answering so late!

**Comment by Hagebuttentee** [comment]
> The frame drops are purely visually, iirc blujay said si himself.

Yeah I'm fully aware of this. However even purely visual framedrops are very distracting from the game and especially in fighting games this is suboptimal. Also the fact that the drops are so frequent is making it far worse than it would be if it were just rare appearences. It would be better if it was like in the seperat build of the mod where it is just one file you cam remove and add as you wish rather than being integrated into HDR as it is right now (or maybe it is a different file entirely which I justndo not find). Overall it makes the game look laggy and somewhat unfinished. I don't want to flame Blu-Dev or anyone else working on the project, I highly respect you people, but I think in its current state the input delay mod is not optimised enough to be integrated into HDR like this. Also sorry for replying so late!

**Comment by Z0niC101** [comment]
On my end, it's very rare because I have a overclock switch.

**Comment by kalomaze** [comment]
Fwiw, overclocking CPU + GPU clocks to safe limits fully resolves the issue. If you're uncomfortable with that, get the 720p handheld mode for docked patch here along with SaltyNX and the Tesla menu for switching to it in real time.

https://github.com/masagrator/ReverseNX-RT

**Comment by kalomaze** [comment]
It also is notable that Pokemon Stadium 2 in its current state has been unoptimized for a while, which makes framedrops most prominent there. It may be smart to do a hotfix with the remade PS2 from Silent & Willy
https://gamebanana.com/mods/456120

**Comment by Hagebuttentee** [comment]
Thank you very much kalomaze, I will try those things.

**Comment by Hagebuttentee** [comment]
> 

What are safe limits?

**Comment by Hagebuttentee** [comment]
> Fwiw, overclocking CPU + GPU clocks to safe limits fully resolves the issue. If you're uncomfortable with that, get the 720p handheld mode for docked patch here along with SaltyNX and the Tesla menu for switching to it in real time.
> 
> https://github.com/masagrator/ReverseNX-RT

What are safe limits?

**Comment by Z0niC101** [comment]
I have mine set to max gou cpu and ram clock speeds for dock only

**Comment by Hagebuttentee** [comment]
> I have mine set to max gou cpu and ram clock speeds for dock only

And this is safe?

**Comment by WuBoytH** [comment]
If you have the normal version of sys-clk, it will only allow you to overclock the switch to safe frequencies.

**Comment by Hagebuttentee** [comment]
> If you have the normal version of sys-clk, it will only allow you to overclock the switch to safe frequencies.

Sorry for answering so late but thank you very much, that pretty much solves it. :)

**Comment by Hagebuttentee** [comment]
Hey, I last commented quiet a while ago. As far as I am aware this issue is still not resolved. You can bypass it by overclocking to the maximum save frequencies, which are the maximum available frequencies in the normal version of sys-clk. While this is a consistant way to avoid the framedrops it should certainly not be required to bypass such a detrimental issue. Some people might not feel well doing this and regard it as unsafe. In my opinion this issue still needs to be resolved either by being able to disable it, fixing the framedrops or removing this patch entirely. This is not meant to be an offense or anything, I just wanted to make aware of this issue again, since my last comment made this issue seem concluded, which it is not, it is just bypassable by other mods.

Thank you for your understanding!
